lib/kafka_ex/protocol/produce.ex

```diff
@@ -53,8 +53,14 @@ defmodule KafkaEx.Protocol.Produce do
     - value: is the message to be written to Kafka logs.
```

Comment:
What about header's doc?

```suggestion
    - value: is the message to be written to Kafka logs.
    - headers: is a list of keys and values that adds metadata to messages
```

Reply:
Indeed